airflow/models/taskinstance.py

```diff
@@ -187,6 +187,191 @@ class TaskReturnCode(Enum):
     """When task exits with deferral to trigger."""
 
 
+@internal_api_call
+@provide_session
+def _merge_ti(ti, session: Session = NEW_SESSION):
+    session.merge(ti)
+    session.commit()
+
+
+@internal_api_call
+@provide_session
+def _add_log(
+    event,
+    task_instance=None,
+    owner=None,
+    owner_display_name=None,
+    extra=None,
+    session: Session = NEW_SESSION,
+    **kwargs,
+):
+    session.add(
+        Log(
+            event,
+            task_instance,
+            owner,
+            owner_display_name,
+            extra,
+            **kwargs,
+        )
+    )
+
+
+def _run_raw_task(
```

Comment:
Can we make a slightly different name for this method in order to better distinguish it from `TaskInstance._run_raw_task()`? Else this might be confusion between class and module method.
Proposal:
```suggestion
def _run_ti_raw_task(
```